style.css

```diff
@@ -592,164 +519,195 @@ button:hover {
-  order: 0;
-  flex-grow: 0;
+  padding: 64px 24px;
+  gap: 58px;
+  background-image: url(/images/form.png);
+  background-repeat: no-repeat;
+  background-position-y: bottom;
+  background-position-x: right;
+  position: absolute;
+  height: 879px;
+  left: 0;
+  top: 5483px;
 }
 
-.card-works > .skills {
-  width: 337px;
-  height: 32px;
-  font-family: 'Inter', sans-serif;
+.form-text {
+  width: 327px;
+  height: 363px;
+  font-family: "Inter", sans-serif;
   font-style: normal;
-  font-weight: 600;
-  font-size: 24px;
-  line-height: 32px;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
   text-align: center;
-  color: #344563;
+  color: #172b4d;
   flex: none;
   order: 0;
   flex-grow: 0;
-}
-
-.card-works > .skills-tag {
   display: flex;
+  flex-direction: column;
   align-items: center;
-  padding: 12px;
-  width: 33px;
-  height: 20px;
-  background: #fff;
-  border-radius: 8px;
+  padding: 0;
+  gap: 24px;
+  margin-top: 128px;
 }
 
-.card-works > .skills-tag li {
+.form {
   display: flex;
-  justify-content: space-between;
+  flex-direction: column;
   align-items: center;
-  padding: 12px;
-  width: 53px;
-  height: 20px;
-  margin-top: 50px;
-  margin-left: 5px;
-  background: #fff;
-  border-radius: 8px;
+  padding: 0;
+  gap: 24px;
+  width: 327px;
+  height: 330px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  justify-content: center;
+  margin-top: 170px;
 }
 
-#capybara {
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-  padding: 12px;
-  width: 69px;
-  height: 10px;
-  margin-top: 80px;
-  margin-left: 80px;
-  background: #fff;
-  border-radius: 8px;
-  color: #36b37e;
+.text-form {
+  width: 327px;
+  height: 48px;
   flex: none;
   order: 0;
   flex-grow: 0;
-}
-
-#Selenium {
+  left: 16px;
+  right: 16px;
+  top: calc(50% - 18px / 2);
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
   display: flex;
-  flex-direction: row;
   align-items: center;
-  padding: 12px;
-  width: 69px;
-  height: 10px;
-  margin-top: 80px;
-  margin-left: 10px;
+  color: #6b778c;
+  box-sizing: border-box;
+  bottom: 0;
   background: #fff;
-  border-radius: 8px;
-  color: #36b37e;
+  border: 1px solid #d0d9d4;
+  border-radius: 4px;
+}
+
+.email {
+  width: 327px;
+  height: 48px;
   flex: none;
   order: 1;
   flex-grow: 0;
-}
-
-#terminal {
+  left: 16px;
+  right: 16px;
+  top: calc(50% - 18px / 2);
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
   display: flex;
-  flex-direction: row;
   align-items: center;
-  padding: 12px;
-  width: 60px;
-  height: 10px;
-  margin-left: 135px;
-  margin-top: 40px;
+  color: #6b778c;
+  box-sizing: border-box;
+  bottom: 0;
   background: #fff;
-  border-radius: 8px;
-  color: #36b37e;
+  border: 1px solid #d0d9d4;
+  border-radius: 4px;
+}
+
+.textarea {
+  width: 327px;
+  height: 114px;
   flex: none;
```

Comment:
- [OPTIONAL] Just a tip 💡 There is a property in CSS called "resize" that you can add in the text area to prevent the user from resizing the text area horizontally by adding "resize: vertical" or "resize: unset" to prevent it from resizing in both directions 😅💯 [Link](https://www.w3schools.com/cssref/css3_pr_resize.asp)